src/main/resources/templates/welcome.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome to GilfGang website</title>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/globe-alt.css' rel='stylesheet'>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/arrow-right-r.css' rel='stylesheet'>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/arrow-left-r.css' rel='stylesheet'>
+    <link rel="stylesheet" type="text/css" href="/static/styles.css">
+</head>
+<body>
+<div class="dropdown">
+    <i class="gg-globe-alt" id="globe-icon"></i>
+    <div class="dropdown-content" id="dropdown-content">
+        <a th:href="@{${httpServletRequest.requestURI} + '?page=' + ${currentPage} + '&lang=en'}">English</a>
+        <a th:href="@{${httpServletRequest.requestURI} + '?page=' + ${currentPage} + '&lang=sv'}">Svenska</a>
+    </div>
+</div>
+<script src="/static/script.js"></script>
```

Comment:
```suggestion
<script src="/static/script.js"></script>
```
I think we should place it in header since it is part of the setup and right now it is mixed with the content, makes no difference but I do think it is cleaner to keep the setup from the content.  ```<script src="/static/script.js" defer></script>``` We should use defer keyword to improve performance. [Defer](https://javascript.info/script-async-defer#defer:~:text=The%20defer%20attribute%20tells,but%20with%20defer%3A)